core/src/main/java/hudson/init/impl/InstallUncaughtExceptionHandler.java

```diff
@@ -20,11 +20,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class InstallUncaughtExceptionHandler {
+
+    private static final Logger LOGGER = Logger.getLogger(InstallUncaughtExceptionHandler.class.getName());
+
     @Initializer
     public static void init(final Jenkins j) throws IOException {
         CompressionFilter.setUncaughtExceptionHandler(j.servletContext, new UncaughtExceptionHandler() {
             @Override
             public void reportException(Throwable e, ServletContext context, HttpServletRequest req, HttpServletResponse rsp) throws ServletException, IOException {
+                if (rsp.isCommitted()) {
+                    LOGGER.log(Level.WARNING, null, e);
```

Comment:
Maybe makes sense to explicitly say that the response is already committed in the log message